instance/instance.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/cozy/cozy-stack/couchdb"
 	"github.com/cozy/cozy-stack/couchdb/mango"
 	"github.com/cozy/cozy-stack/vfs"
+	"github.com/dchest/uniuri"
+	scrypt "github.com/elithrar/simple-scrypt"
```

Comment:
ok, I will copypaste some of simple-scrypt into cozy-stack. 
It handles all the complexity like the salt you mentionned below.